tests/test_farmer.py

```
@@ -0,0 +1,172 @@
+            "password": "testpassword123",
+            "user_type": "FARMER"
+        }
+        
+        response = await client.post("/api/users/register", json=user_data)
+        assert response.status_code == 201
+        
+        user_response = response.json()
+        user_id = user_response["id"]
+
+        # Create farmer profile
+        farmer_data = {
```

Ruff (0.12.2)

Local variable `farmer_data` is assigned to but never used

Remove assignment to unused variable `farmer_data`

(F841)